apps/prairielearn/src/lib/externalGrader.sql

```diff
@@ -10,7 +10,7 @@ FROM
   LEFT JOIN submissions AS s ON (s.id = gj.submission_id)
   LEFT JOIN variants AS v ON (v.id = s.variant_id)
   LEFT JOIN questions AS q ON (q.id = v.question_id)
-  LEFT JOIN pl_courses AS c ON (c.id = v.course_id)
+  LEFT JOIN pl_courses AS c ON (c.id = q.course_id)
```

Comment:
External grader should be passed the `question_course` to do the grading, not the `variant_course`!

Is there any reason we also need to grab the `variant_course` here? 

In a lot of places I am grabbing the info for both courses, so the `variant_course` can be used to report issues, but I can't find anywhere that issues related to external grading are being reported.

Reply:
Huh, that might be a separate problem if we aren't reporting failures as issues.